### PR TITLE
Update `cypress_repro.md` template

### PR DESCRIPTION
**Pull request by nemanjaglumac**
### Status
PENDING REVIEW

### What does this PR accomplish?
- Updates Cypress repro template
- Removes Prettier's formatting from this document which was creating unnecessary empty lines
- Updates certain sections and descriptions based on the previous heavy usage

### Additional notes:
- It is very helpful to include screenshots at the bottom of your Cypress repro PR. It shows the state of the app at the moment of failure and it makes reviewer's life easier.
- It is also helpful to include an alternative version of the screenshot/state of the app (for local testing only). I call these a "sanity check" screenshots.
- I removed "Questions" section. All conversation related to the code should be in the PR comments, and comments on a specific line(s) of code.
- Feel free to also include GIFs, and/or video screenshots

### Examples:
- Sanity check screenshot included:
    - https://github.com/metabase/metabase/pull/14536
    - https://github.com/metabase/metabase/pull/14530
- Screenshots focusing on specific parts of the UI (Cypress test-runner, app UI...)
    - https://github.com/metabase/metabase/pull/14498

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/cypress_repro.md [hunk 1]
```diff
@@ -1,31 +1,25 @@
...
 
 ### What does this PR accomplish?
-
 - Reproduces #XXXXX
 
 ### How to test this manually?
-
 - `yarn test-cypress-open`
-- `relative/path/to/the/file` _(optionally, include the line number on which the test starts)_
+- `relative/path/to/the/file`
```
Comment (nemanjaglumac) [inline .github/PULL_REQUEST_TEMPLATE/cypress_repro.md:9]: There are some duplicate spec names living under different folders. That's why it's helpful to include the whole path.

Hint: 
- You can paste that same path in Cypress UI's search field.

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1 @@
+cypress_repro.md
```
Comment (tsmacdonald) [inline .prettierignore:1]: `prettier` is looking inside of `.github`? I would have thought it was just looking in `frontend` and `enterprise`

```
// package.json:190
    "prettier": "prettier --write '{enterprise/,}frontend/**/*.{js,jsx,css}'",
```

File: .github/PULL_REQUEST_TEMPLATE/cypress_repro.md [hunk 3]
```diff
@@ -1,31 +1,25 @@
...
 > _(For still unfixed bug)_
-- Replace `it.skip()` with `it.only()` to test this in isolation
+- Replace `it.skip()` with `it.only()` to run test in isolation
```
Comment (tsmacdonald) [inline .github/PULL_REQUEST_TEMPLATE/cypress_repro.md:11]:
```suggestion
- Replace `it.skip()` with `it.only()` to run the test in isolation
```

**Review by tsmacdonald — APPROVED** [review]

File: .prettierignore (hunk 2 above)
Comment (nemanjaglumac) [reply .prettierignore:1]: Oopsy... that would be my `"editor.formatOnSave": true,`, I guess 🥴
Thanks for spotting this. I'll remove `.prettierignore`.

**Pull request merged**